apps/astarte_data_updater_plant/test/astarte_data_updater_plant/data_updater/payloads_decoder_test.exs

```diff
@@ -192,6 +192,11 @@ defmodule Astarte.DataUpdaterPlant.PayloadsDecoderTest do
     assert PayloadsDecoder.safe_inflate(compressed) == :error
   end
 
+  test "zlib inflate does not crash with a payload that is not zlib deflated" do
+    random_bytes = :crypto.strong_rand_bytes(System.unique_integer([:positive]))
```

Comment:
There's a non-zero chance that this produces a valid a valid zlib payload, and also `System.unique_integer` might get very big if this test is executed last in the test-suite. Replace this with a fixed non-valid payload.